packages/block-editor/src/components/block-list/use-in-between-inserter.js

```diff
@@ -40,7 +40,7 @@ export function useInBetweenInserter() {
 			}
 
 			function onMouseMove( event ) {
-				if ( openRef.current ) {
+				if ( openRef === undefined || openRef.current ) {
```

Comment:
I think this would work too.
```suggestion
				if ( openRef?.current ) {
```

Reply:
It was my first commit :D 

However, with this code, the function will not return at line 44, but will continue with the logic.

BTW, I don't have a strong opinion.

Reply:
These are different checks:

1. The proposed logic in the PR is bail on `openRef === undefined` OR if `openRef.current` is truthy, which I'm not sure is correct.
2. @cbravobernal's suggestion bails on openRef truthy, but openRef === undefined keeps going.

@gigitux Could you explain why the handler should bail on `openRef === undefined`?

Reply:
> It was my first commit :D
> 
> However, with this code, the function will not return at line 44, but will continue with the logic.
> 
> BTW, I don't have a strong opinion! What do you think?

Agh, we made the same mistake then 😆 

Return means not showing the in-between inserter. openRef seems to be a reference of where the next block should be insterted.

So, if the reference of the next position to be insterted does not exist (is undefined), the indicator should be hidden.

Reply:
@vcanales Good point! Thanks for clarifying this! My first intuition was the right one. Sometimes overthinking isn't a good idea 😅

Reply:
Oh now I understand, thanks for the explanation @cbravobernal
@gigitux I think `openRef === undefined || openRef.current` is the correct check, since we don't want to show the inserter when the next ref is undefined.

Reply:
This is from where I understood that returning is hiding the inserter btw:
https://github.com/WordPress/gutenberg/blob/2d34e2aaa56e52c379b01bed2df2344ba01c9bb4/packages/block-editor/src/components/block-list/use-in-between-inserter.js#L132